lighthouse-core/lib/dependency-graph/simulator/network-analyzer.js

```diff
@@ -303,52 +303,44 @@ class NetworkAnalyzer {
    * is unavailable.
    *
    * @param {LH.Artifacts.NetworkRequest[]} records
-   * @param {object} [options]
-   * @return {Map<string, !NetworkAnalyzer.Summary>}
+   * @param {NetworkAnalyzer.RTTEstimateOptions} [options]
+   * @return {Map<string, NetworkAnalyzer.Summary>}
    */
   static estimateRTTByOrigin(records, options) {
-    options = Object.assign(
-      {
-        // TCP connection handshake information will be used when available, but for testing
-        // it's useful to see how the coarse estimates compare with higher fidelity data
-        forceCoarseEstimates: false,
-        // coarse estimates include lots of extra time and noise
-        // multiply by some factor to deflate the estimates a bit
-        coarseEstimateMultiplier: 0.3,
-        // useful for testing to isolate the different methods of estimation
-        useDownloadEstimates: true,
-        useSendStartEstimates: true,
-        useHeadersEndEstimates: true,
-      },
-      options
-    );
+    const {
+      forceCoarseEstimates = false,
+      coarseEstimateMultiplier = 0.3,
```

Comment:
FWIW, I think we lose a little value by moving the explanation away from the default on this one

Reply:
> FWIW, I think we lose a little value by moving the explanation away from the default on this one

Yeah, sounds good. We should maybe move it to a top-level constant (with the comment), but just moved the comment back into place for now